src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2027,6 +2027,11 @@ End Class
                               </Document>)
                 state.SendTypeChars("Task")
                 Await state.WaitForAsynchronousOperationsAsync()
+                ' WaitForAsynchronousOperationsAsync actually does not wait for async completion to complete
+                ' TODO need to re-design WaitForAsynchronousOperationsAsync to handle async completion as well.
+                ' https://github.com/dotnet/roslyn/issues/33379 to fix this.
+                ' As a workaround to wait for async completion to complete, we need to get items.
+                state.GetCompletionItems()
```

Comment:
Why not just push this into WaitForAsyncOperationsAsync()? Is the assumption that this one is the only place this needs the update because the rest of them are asserting items which *does* do an implicit wait there?

Reply:
Yes, it is actually the only place. 

I tried to move it into WaitForAsyncOperationsAsync(). It required much more changes to the test infrastructure to avoid duplicate calls of state.GetCompletionItems().